docs/sphinx/platforms.rst

```diff
@@ -0,0 +1,64 @@
+Supported platforms
+===================
+
+The OME C++ software components are written using portable,
+cross-platform C++.  The components and their dependencies are tested
+on a daily basis and are proven to work on a set of core platforms
+(they build without error and pass their unit and integration tests).
+Builds are provided for each release for these platforms.  The support
+status for these is as follows:
+
++---------------+--------------+--------------+--------------+-------------+
+|               |           Version           |         Build type         |
++---------------+--------------+--------------+--------------+-------------+
+| System        | Recommended  | Minimum      | Platform     | Super-Build |
++===============+==============+==============+==============+=============+
+| CentOS        | 7            | 6            | Unsupported* | Supported   |
++---------------+--------------+--------------+--------------+-------------+
+| Debian        | 8            | 7            | Unsupported  | Unsupported |
++---------------+--------------+--------------+--------------+-------------+
+| FreeBSD       | 11           | 10           | Supported†   | Supported   |
++---------------+--------------+--------------+--------------+-------------+
+| MacOS X       | 10.11        | 10.9         | Supported‡   | Supported   |
++---------------+--------------+--------------+--------------+-------------+
+| Ubuntu        | 16.04        | 14.04        | Supported    | Supported   |
++---------------+--------------+--------------+--------------+-------------+
+| Windows       | VS2015       | VS2013       | Unsupported§ | Supported   |
++---------------+--------------+--------------+--------------+-------------+
+
+\*
+  CentOS does not provide all the prequired libraries; CentOS 6
+  provides a broken version of Boost and an obsolete libtiff, so using
+  the Super-Build or building new versions of the needed libraries is
```

Comment:
we usually don't capitalize "super-build", may want to ensure the docs are consistent in this